tests/Resources/Page/ODH/JupyterHub/HighAvailability.robot

```diff
@@ -71,3 +74,24 @@ Verify JupyterHub Deployment
     Should Not Be Equal As Strings  ${leader}  None
     Should Be Equal As Strings  ${leader-found}  True
     Should Be Equal As Integers  ${standby}  2
+
+Wait Until JH Deployment Is Ready
+    [Arguments]   ${retries}=50
+    FOR  ${index}  IN RANGE  0  1+${retries}
+        @{JH} =  OpenShiftCLI.Get  kind=Pod  namespace=redhat-ods-applications  label_selector=deploymentconfig = jupyterhub
+        ${containerNames} =  Create List  jupyterhub  jupyterhub-ha-sidecar
+        ${jh_status}=    Run Keyword And Return Status    Verify JupyterHub Deployment  ${JH}  3  2  ${containerNames}
+        Exit For Loop If    $jh_status == True
+        Sleep    0.5
+    END
+    IF    $jh_status == False
+        Fail    Jupyter Deployment not ready. Checks the logs
+    END
+    Sleep   1
+
+Rollout JupyterHub
```

Comment:
Is rollout equivalent of Scale Deployment down to zero and scale it back?  We have a keyword for that in tests/Resources/ODS.robot 

Maybe you could use that or add yours to the same file

Reply:
hm I think they are technically different, because rollout gradually restart the pods. It could be that the final result is similar/the same. In any case, the kw `Star Rollout` does it in one line so if I have to choose i pick `Start Rollout`